client/src/components/NavBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "../styles/main.css";
+import { NavLink } from "react-router-dom";
+
+function NavBar(props) {
+  //evt handler for clickable nav links, refers to linkDisabled
+  const navLinkClick = (evt) => {
+    //check for if the user has input any text into username
+    if (props.linkDisabled === true) {
+      evt.preventDefault();
+      alert("Please create a username before choosing a room!")
```

Comment:
There is currently a minor bug relating to this statement. If a user inputs a name, and presses enter on the keyboard, the name goes into the URL, but when they try to click on the room, it tells them they need to create a username.